__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const app = require("../app.js");
+const connection = require("../db/connection.js");
+const data = require("../db/data/test-data/index");
+const seed = require("../db/seeds/seed.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  connection.end();
+});
+
+describe("app", () => {
+  describe("/api", () => {
+    test("200: GET responds with a message of connection", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Connected");
```

Comment:
This is an unnecessary test/endpoint, if it's responding with data like in the next test then there must be a connection.